curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -628,3 +628,494 @@ Which of the following is the default value of the `position` property?
+
+#### --distractors--
+
+`clear: all-sides;`
+
+---
+
+`clear: both-sides;`
+
+---
+
+`clear: left-right;`
+
+#### --answer--
+
+`Places the element on the right side of its container, allowing inline content to wrap around it` 
+
+### --question--
+
+#### --text--
+
+What happens when you apply `float: left;` to an element in CSS?
+
+#### --distractors--
+
+`It centers the element horizontally inside its container`  
+
+---
+
+`It aligns the element at the bottom of the container`  
+
+---
+
+`It moves the element to the right side of the container`
+
+#### --answer--
+
+`It removes the element from normal flow and positions it to the left, allowing text to wrap around it`
+
+### --question--
+
+#### --text--
+
+What CSS property can be used to move an element below floated elements?
+
+#### --distractors--
+
+`align`  
+
+---
+
+`offset`
+
+---
+
+`position`
+
+#### --answer--
+
+`clear`
+
+### --question--
+
+#### --text--
+
+Which CSS snippet correctly applies a clearfix hack to fix float collapsing?
+
+#### --distractors--
+
+```css
+.clearfix::before {
+  content: "";
+  display: table;
+  clear: both;
+}
+```
+
+---
+
+```css
+.clearfix::after {
+  content: "";
+  display: inline;
+  clear: left;
+}
+```
+
+---
+
+```css
+.clearfix {
+  content: "";
+  display: block;
+}
+```
+
+#### --answer--
+
+```css
+.clearfix {
+  content: "";
+  display: block;
+  clear: both;
+}
+```
+
+### --question--
+
+#### --text--
+
+In static positioning, how are elements arranged on the page?
+
+#### --distractors--
+
+`From right to left and bottom to top`
+
+---
+
+`Randomly depending on screen size`
+
+---
+
+`Stacked based on their z-index by default`
+
+#### --answer--
+
+`From top to bottom and left to right, following normal document flow`
+
+### --question--
+
+#### --text--
+
+What effect does position: relative have on an element?
+
+#### --distractors--
+
+`It removes the element from the document flow`
+
+---
+
+`It makes the element behave like fixed positioning`
+
+---
+
+`It prevents the element from moving`
+
+#### --answer--
+
+`It allows offsetting an element while keeping it in the normal document flow`
+
+### --question--
+
+#### --text--
+
+What best describes position: absolute in CSS?
+
+#### --distractors--
+
+`It keeps the element fixed in place when scrolling`
+
+---
+
+`It stacks the element above all others automatically`
+
+---
+
+`It restricts the element inside its parent container always`
+
+#### --answer--
+
+`It removes the element from the normal flow and positions it relative to the nearest positioned ancestor`
+
+### --question--
+
+#### --text--
+
+What happens when an element is given position: fixed?
+
+#### --distractors--
+
+`It scrolls along with the page`
+
+---
+
+`It remains in normal flow but moves slightly with scrolling`
+
+---
+
+`It positions relative to its parent element`
+
+#### --answer--
+
+`It is positioned relative to the viewport and stays fixed during scrolling`
```

Comment:
these don't need to be wrapped in backticks. please fix.
plus they need periods.